### PR TITLE
Introduce image_list performance benchmark that runs on jit(debug) build.

### PR DESCRIPTION
**Pull request by aam**
We want to track performance of debug build because ongoing work on reducing the cost of spanwing new isolate will impact jit(debug) more than it will aot(release) configurations.

cc @mkustermann

### CODE REVIEW

File: dev/devicelab/bin/tasks/image_list_jit_reported_duration.dart [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Future<void> main() async {
+  await task(ReportedDurationTest(
+    ReportedDurationTestFlavor.debug,
+    '${flutterDirectory.path}/examples/image_list',
+    'lib/main.dart',
+    'com.example.image_list',
+    RegExp(r'===image_list=== all loaded in ([\d]+)ms.'),
+  ).run);
+}
```
Comment (goderbauer) [inline dev/devicelab/bin/tasks/image_list_jit_reported_duration.dart]: nit: end with newline?

File: dev/devicelab/manifest.yaml [hunk 2]
```diff
@@ -352,7 +352,14 @@ tasks:
 
   image_list_reported_duration:
     description: >
-      Measures image loading performance.
+      Measures image loading performance on release(aot) build.
```
Comment (goderbauer) [inline dev/devicelab/manifest.yaml]: nit: space before (aot)?

File: dev/devicelab/manifest.yaml [hunk 3]
```diff
@@ -352,7 +352,14 @@ tasks:
...
+    stage: devicelab
+    required_agent_capabilities: ["linux/android"]
+    flaky: true
+
+  image_list_jit_reported_duration:
+    description: >
+      Measures image loading performance on debug(jit) build.
```
Comment (goderbauer) [inline dev/devicelab/manifest.yaml]: nit: space before (jit)?

**Review by goderbauer — APPROVED** [review]
LGTM

File: dev/devicelab/bin/tasks/image_list_jit_reported_duration.dart (hunk 1 above)
Comment (aam) [reply dev/devicelab/bin/tasks/image_list_jit_reported_duration.dart]: Done

File: dev/devicelab/manifest.yaml (hunk 2 above)
Comment (aam) [reply dev/devicelab/manifest.yaml]: Done

File: dev/devicelab/manifest.yaml (hunk 3 above)
Comment (aam) [reply dev/devicelab/manifest.yaml]: Done

**Pull request merged**